notebooks/docker_and_kubernetes/labs/3_k8s_hello_node.ipynb

```diff
@@ -26,6 +26,19 @@
     "<img src='../assets/k8s_hellonode_overview.png' width=\"500\"/>"
```

Comment:
<p>It might be better to note that this command has to be run in the terminal. If you run it for the first time, it prompts for an Y/N question and you can't respond to that in the jupyter cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/328/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Good catch! Thanks @sanjanalreddy 
I now modified the script so that it no longer prompts for Y/N.